docs/develop-load-manager.md

```diff
@@ -1,149 +1,45 @@
-2. Using the `pulsar-admin` tool. Here's an example:
+- Modular load manager, implemented in [`ModularLoadManagerImpl`](https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java), which is a flexible alternative to the [`SimpleLoadManagerImpl`](https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java).
 
-   ```shell
-   pulsar-admin update-dynamic-config \
-   --config loadManagerClassName \
-   --value org.apache.pulsar.broker.loadbalance.impl.ModularLoadManagerImpl
-   ```
-
-   You can use the same method to change back to the original value. In either case, any mistake in specifying the load manager will cause Pulsar to default to `SimpleLoadManagerImpl`.
-
-## Verification
-
-There are a few different ways to determine which load manager is being used:
-
-1. Use `pulsar-admin` to examine the `loadManagerClassName` element:
-
-   ```shell
-   bin/pulsar-admin brokers get-all-dynamic-config
-   {
-    "loadManagerClassName" : "org.apache.pulsar.broker.loadbalance.impl.ModularLoadManagerImpl"
-   }
-   ```
-
-   If there is no `loadManagerClassName` element, then the default load manager is used.
-
-2. Consult a ZooKeeper load report. With the module load manager, the load report in `/loadbalance/brokers/...` will have many differences. For example, the `systemResourceUsage` sub-elements (`bandwidthIn`, `bandwidthOut`, etc.) are now all at the top level. Here is an example load report from the module load manager:
-
-   ```json
-   {
-     "bandwidthIn": {
-       "limit": 10240000.0,
-       "usage": 4.256510416666667
-     },
-     "bandwidthOut": {
-       "limit": 10240000.0,
-       "usage": 5.287239583333333
-     },
-     "bundles": [],
-     "cpu": {
-       "limit": 2400.0,
-       "usage": 5.7353247655435915
-     },
-     "directMemory": {
-       "limit": 16384.0,
-       "usage": 1.0
-     }
-   }
-   ```
-
-   With the simple load manager, the load report in `/loadbalance/brokers/...` looks like this:
-
-   ```json
-   {
-     "systemResourceUsage": {
-       "bandwidthIn": {
-         "limit": 10240000.0,
-         "usage": 0.0
-       },
-       "bandwidthOut": {
-         "limit": 10240000.0,
-         "usage": 0.0
-       },
-       "cpu": {
-         "limit": 2400.0,
-         "usage": 0.0
-       },
-       "directMemory": {
-         "limit": 16384.0,
-         "usage": 1.0
-       },
-       "memory": {
-         "limit": 8192.0,
-         "usage": 3903.0
-       }
-     }
-   }
-   ```
-
-3. The command-line [broker monitor](reference-cli-tools.md) will have a different output format depending on which load manager implementation is being used.
-
-   Here is an example from the modular load manager:
-
-   ```
+- Extensible load manager, implemented in 
```

Comment:
@Demogorgon314 as we discussed just now, can you provide technical input for this?